docs/JestObjectAPI.md

```diff
@@ -428,9 +428,11 @@ When this API is called, all pending micro-tasks that have been queued via `proc
 
 ### `jest.runAllTimers()`
 
-Exhausts the **macro**-task queue (i.e., all tasks queued by `setTimeout()`, `setInterval()`, and `setImmediate()`).
+Exhausts both the **macro**-task queue (i.e., all tasks queued by `setTimeout()`, `setInterval()`, and `setImmediate()`) and the **micro**-task queue (usually interfaced in node via `process.nextTick`).
 
-When this API is called, all pending "macro-tasks" that have been queued via `setTimeout()` or `setInterval()` will be executed. Additionally if those macro-tasks themselves schedule new macro-tasks, those will be continually exhausted until there are no more macro-tasks remaining in the queue.
+When this API is called, all pending macro-tasks that have been queued via `setTimeout()` or `setInterval()` will be executed. If those macro-tasks themselves schedule new macro-tasks, those will be continually exhausted until there are no more macro-tasks remaining in the queue.
+
+Additionally, all pending micro-tasks that have been queued via `process.nextTick` will be executed. If those micro-tasks themselves schedule new micro-tasks, those will be continually exhausted until there are no more micro-tasks remaining in the queue.
```

Comment:
Seems like basically a copy-paste of the paragraph above.

Could we drop this new paragraph, and expand the one before it to be:

"When this API is called, all pending macro-tasks and micro-tasks will be executed. If those tasks themselves schedule new tasks, those will be continually exhausted until there are no more tasks remaining in the queue."

No need to repeat the info about `setTimeout` etc